lib/ansible/executor/powershell/module_manifest.py

```diff
@@ -294,6 +294,7 @@ def _create_powershell_wrapper(b_module_data, module_path, module_args,
         exec_manifest["actions"].insert(0, 'async_wrapper')
         exec_manifest["async_jid"] = str(random.randint(0, 999999999999))
         exec_manifest["async_timeout_sec"] = async_timeout
+        exec_manifest["async_named_pipe_timeout"] = C.config.get_config_value('DEFAULT_WIN_ASYNC_STARTUP_TIMEOUT')
```

Comment:
```suggestion
        exec_manifest["async_named_pipe_timeout"] = C.DEFAULT_WIN_ASYNC_STARTUP_TIMEOUT
```

You can just access the config value directly, the only reason why you would want to use `get_config_value()` is if you wanted to inject extra vars into the lookup which isn't relevant here.